base/src/com/thoughtworks/go/logging/LogConfigurator.java

```diff
@@ -82,13 +80,35 @@ protected void initializeFromXMLResource(URL resource) {
     }
 
     protected void initializeFromPropertiesFile(File log4jFile) {
-        watchDog = new PropertyWatchdog(log4jFile.getAbsolutePath());
-        watchDog.setDelay(DELAY);
+        CustomFileWatchdog watchdog = new CustomFileWatchdog(log4jFile) {
```

Comment:
Wasn't able to startup Go-server using zip installer after this change, it fails with 
```
Caused by: java.lang.NoClassDefFoundError: com/thoughtworks/go/logging/LogConfigurator$1
	at com.thoughtworks.go.server.util.GoLauncher.main(GoLauncher.java:41)
	... 6 more
```

Reply:
Fixed